### DISCUSSION TITLE
Sick of chia time to drop this rubbish project

### DISCUSSION THREAD

**Discussion by waynedavies045** [Chia-Network/chia-blockchain#16768] [Support]
Why are you not on any proper exchanges ? Is this a scam ?
I have tried to swap tokens and lost tokens as your clawback doesnt even work.
I have spent £2,000 on hdd's and done 1,000's hrs plotting to make 80p a day what a joke honeygain makes more for free
So how do I sell my tokens so I can get away from this rubbish scheme
6yrs roi omg
Be lucky if you last 1 year
My worst crypto venture so far

**Comment by BrandtH22** [comment]
If you are interested in troubleshooting the clawback issues I would first start by upgrading to the latest version (2.1.1) of chia and running the below commands in PowerShell (windows) or terminal (mac/linux) to force the clawback transaction:

1. Identify the clawback transaction id by running `chia wallet get_transactions --clawback`
2. Use the transaction id in this command and for windows run `chia rpc wallet spend_clawback_coins '{"coin_ids":["TRANSACTION_ID"], "force":true, "fee":0}'` | for linux run `chia rpc wallet spend_clawback_coins '{"coin_ids":["TRANSACTION_ID"], "force":true, "fee":0}'``

Make sure to replace the TRANSACTION_ID in the second command with the resulting transaction id from the first command.

Give it some time and you should see the clawback in the wallet.

**Comment by waynedavies045** [comment]
Invalid REQUEST JSON: Expecting property name enclosed in double quotes: line 1 column 2 (char 1)   ???

**Reply by BrandtH22** [reply]
Hey, based on the error I am assuming that you are on windows and will need to rerun the second command with the double quotes escaped similar to:
`chia rpc wallet spend_clawback_coins '{\"coin_ids\":[\"TRANSACTION_ID\"], \"force\":true, \"fee\":0}'`

I am sorry for any confusion as it seems my initial message did not have the quotes properly escaped for the windows command

**Reply by BrandtH22** [reply]
Note - I have removed hajes comment and reported the user for their behavior.

**Reply by waynedavies045** [reply]
That worked.. Clawed it back in minutes.. Thanks for your help and time. Shows there's still a few good-uns out there. Wish you all the best in all you do.. Thanks again

**Reply by BrandtH22** [reply]
Thank you for the confirmation and the kind words. I wish you the best also and am available on discord if any other questions arise.

I am going to close this discussion ticket but feel free to reopen it or create a new one if issues arise.

**Comment by hajes** [comment]


**Reply by waynedavies045** [reply]
Not blamed others
Not in poverty
Not whined
Just asked why not on binance or Coinbase etc
And asked how to sell as think its a rubbish project
You reply with no usefull info and make false accusations. GFY

**Reply by hajes** [reply]
Selective blindness, the common issue with not-my-fault syndrome...I wish you luck in your life. There is no useful info for bullshit darling ;-) take it or leave it, nobody forces you to use Chia nor will do work for you :-P

**Reply by waynedavies045** [reply]
Sorry ok I see now there is something wrong with you or you just can't read, I was referred here for help which BrandtH22 did. You just made nasty snide comments and probably think you were really cool.  But it just revealed how sad you really are. So glad I don't know you. Deleted my fake email, add as many snide comments as you like now as I won't see them. If you get a few out of your system you might become a better person in real-life.. Bye-bye "darling"

**Reply by BrandtH22** [reply]
@hajes , this is completely unnecessary.
Please be respectful of others.
